### ISSUE TITLE
Option for multiple photos for some species in App guide

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/ABLE#197]
To provide images to help identification of species, and for sexual dimorphism

@kazlauskis to advise on implications for size of app on phone

**Comment by kazlauskis** [comment]
At the moment, Android app size is 50Mb having 230 photos (out of 500 species). We could add as many photos as you would like, but it gets slightly trickier when the apps go over 100Mb. We would have to start hosting the photos ourselves instead of bundling with the code and uploading to the app stores. Adding another 200 photos would render this task `small` otherwise `medium` to `large`.

**Comment by CrisSevilleja** [comment]
Coming back to this issue, several people asked me to improve the photo App guide of ButterflyCount providing some pictures more and completing all species. I know it is tricky to decide about increasing or not the app size, but I would try to complete at least the species list and maybe later we can include sexual dimorphism and upper/underside pictures. 

Do you see possibilities @kazlauskis  to start hosting the photo ourselves or find another solution to include more pictures? 

People are quite happy with the guide, even experts to have a fast look at species. I have several people happy to provide more pictures.

**Comment by kazlauskis** [comment]
Yes, we can add more photos to the app and two (or maybe more) photos per species sound OK to me.

**Comment by CrisSevilleja** [comment]
ok, can you give me the list of species with a picture? I will prepare the pictures for the remaining species

**Comment by kazlauskis** [comment]
Species with an `Image_width` column value in this spreadsheet have an image.

[species.profiles.csv](https://github.com/BiologicalRecordsCentre/ABLE/files/10665145/species.profiles.csv)

**Comment by Vilius-Stankaitis** [comment]
Hi @CrisSevilleja, 

Any updates?

This is a large ticket to update all missing species photos, maybe we can add thumbnails for the top recorded species it would give the app some freshness.

![Screenshot 2023-05-15 at 09.48.46.png](https://zube.io/files/biologicalrecordscentre/b6d669515bd70bb0746fe0f045aa7348-screenshot-2023-05-15-at-09-48-46.png)

**Comment by CrisSevilleja** [comment]
Hi @Vilius-Stankaitis I was working on this before my holidays and in the end I didn't share it with you. 

I like a lot the idea of using thumbnails for the top recorded species, indeed it would give the app some freshness.

Continuing with the idea of this issue was to add pictures of butterfly species we are missing. I asked some of my butterfly colleagues and coordinators and they completed a bit of the list Karolis gave me. I just share with you a google drive folder where I included the pictures to use on the app for missing species (especially species on remote islands or eastern countries). I don't know how much it will take you to include those pictures. 

I am finishing placing in the family folders, the species name and author on the picture name and avoid all the folders. I will finish it today.

**Comment by CrisSevilleja** [comment]
@kazlauskis did you get my google drive with the pictures? are these pictures included in the app? thanks

**Comment by Vilius-Stankaitis** [comment]
@CrisSevilleja Hi, yes. I have moved "species_pictures_ButterflyCount" to Flumens OneDrive and it is now linked to the app. 

Some species descriptions are still missing, so not all images will be displayed. Also, only "Limenitis reducta" (Spain) and "Euchloe tagis" has two images. 

Report page is updated as well.

This should be available in the next release. 

https://butterflies.app.flumens.io/demo.html
